src/components/Details.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import '../styles/Details.scss';
+import { Link } from 'react-router-dom';
+import logo from '../assets/images/logo.svg';
+import '../styles/Global.scss';
+
+
+class Details extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {},
+      repos: [],
+      loading: true,
+    }
+    this.fetchRepositories = this.fetchRepositories.bind(this);
+    this.getUser = this.getUser.bind(this);
+  }
+
+  componentDidMount() {
+    this.getUser();
+  }
+
+  getUser() {
+    const { match: { params: { id } }, listOfUsers: { items } } = this.props;
+    const foundUser = items.find((user) => user.id === parseInt(id, 10));
+    this.setState({
+      user: foundUser,
+    }, () => this.fetchRepositories(foundUser.repos_url))
+  }
+
+  fetchRepositories(repos_url) {
+    return fetch(`${repos_url}`)
```

Comment:
@marcoglnd, Não seria mais adequado adaptar esta linha para a forma abaixo?
```suggestion
    return fetch(repos_url)
```